keps/sig-node/1539-hugepages/README.md

```diff
@@ -546,6 +560,69 @@ locality guarantees as a feature of QoS.  In particular, pods in the
     - Test case will be added to cri-tools to be used in CRI runtime' test(CI).
     - here: https://github.com/kubernetes-sigs/cri-tools
 
+## Test Plan for HugePageStorageMediumSize
+
+- Promote existing HugePages E2E tests to conformance
+
+## Production Readiness Review Questionnaire for HugePageStorageMediumSize
+### Monitoring requirements
+
+* **How can an operator determine if the feature is in use by workloads?**
+An operator could use hugepages-<size> resource limits and emptydir
+mounts with medium: HugePage-<size> ad described in the Kubernetes
+documentation at https://kubernetes.io/docs/tasks/manage-hugepages/scheduling-hugepages
+
+* **What are the SLIs (Service Level Indicators) an operator can use to determine.
+the health of the service?**
+N/A
+
+* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
+N/A
```

Comment:
Here is my suggestion for how to answer these two questions:

SLIs: Workload performance can be measured by existing system metrics provided by Kubernetes components and e.g. [node_exporter](https://github.com/prometheus/node_exporter).

SLOs: These will be set individually by application developers. This feature allows them to tune the performance of their workloads. See e.g. https://blog.yannickjaquier.com/linux/linux-hugepages-and-virtual-memory-vm-tuning.html

Reply:
Thank you for the suggestion. I've added kube_pod_resource_request and kube_pod_resource_limit metrics to SLIs answer.